src/test/lib/BasicTest.cpp

```diff
@@ -270,3 +270,24 @@ void QuicTestBindConnectionExplicit(_In_ int Family)
         TEST_QUIC_SUCCEEDED(Status);
     }
 }
+
+void QuicTestAddrFunctions(_In_ int Family)
+{
+    QUIC_ADDR SockAddr = {};
```

Comment:
I feel like we should explicitly memset this to `0xFF` first, to make sure the checks below would fail if they aren't setting everything properly.